src/models/opt_decoder.h

```diff
@@ -47,4 +47,4 @@ class OptDecoder : public CommonDecoder<Attention<WeiT, QKPO_Dummy, LayerNorm>,
     LayerNorm finalLN;
 };
 
-REGISTER_MODEL(OptDecoder, gpt)
+REGISTER_MODEL(OptDecoder, opt)
```

Comment:
It should be 'gpt' since the model type in the converted config.ini is gpt.

Reply:
Or update the model type name in the converter to `gpt`

Reply:
Done